src/components/SvgLibraries/shared/SvgLibrary.module.scss

```diff
@@ -10,46 +10,100 @@
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 0 $spacing-05;
+  position: sticky;
+  z-index: z('dropdown');
+  top: 4rem;
+  border-bottom: 1px solid $ui-03;
+  transition: box-shadow $duration--fast-02 ease;
   @include carbon--breakpoint('md') {
     align-items: flex-end;
-    padding: 0;
     flex-direction: row;
   }
 }
 
+.filter-row[data-stuck] {
+  box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.2);
+}
+
 .filter-row :global(.bx--search) {
   @include carbon--breakpoint('md') {
-    width: 60%;
-    margin-right: $spacing-11;
+    width: calc(50% + 1px);
+    border-right: 1px solid $ui-03;
+  }
+  @include carbon--breakpoint('lg') {
+    width: calc(66.666% + 1px);
+    border-right: 1px solid $ui-03;
+  }
+}
+
+.filter-row.pictograms :global(.bx--search) {
+  @include carbon--breakpoint('lg') {
+    width: calc(75% + 1px);
+  }
+}
+
+.filter-row::before {
+  content: '';
+  position: absolute;
+  z-index: z('dropdown');
+  height: 1rem;
+  width: 100%;
+  top: -1rem;
+  background: #f3f3f3;
```

Comment:
Do we wanna use a token for this?

Reply:
@jnm2377 I don't think there is one :( It'd be like gray-05 or something.